specification/migrate/create_from/MigrateCreateFromResponse.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { IndexName } from '@_types/common'
+
+export class Response {
+  body: {
+    acknowledged: boolean
+    index: IndexName
+    shards_acknowledged: boolean
+  }
+}
```

Comment:
I'm wondering, how/why isn't [toXContent](https://github.com/elastic/elasticsearch/blob/859f9c38b5aca969d894db0e79af93671b7a6e1c/x-pack/plugin/migrate/src/main/java/org/elasticsearch/xpack/migrate/action/CreateIndexFromSourceAction.java#L125-L141) used? I know it's not a mistake, as it's confirmed by the YAML tests.

Reply:
Ha, we knew that question would come up, which is why it's in that comment:
```
        /*
         * This only exists for the sake of testing the xcontent parser
         */
```
This is the request, not the response. The response is just `AcknowledgedResponse`. The request accepts xcontent. In order to use AbstractXContentSerializingTestCase, we added this toXContent (although for unrelated reasons we later ripped that out and wrote our own xcontent serialization test, and could have moved this toXContent into the unit test).